templates/function-base/ts/default/package.json

```diff
@@ -13,7 +13,7 @@
     "dependencies": {
         "@azure/functions": "^1.2.2",
         "@microsoft/microsoft-graph-client": "^3.0.0",
-        "@microsoft/teamsfx": "^0.5.0",
+        "@microsoft/teamsfx": "^0.5.2-alpha.594b33666.0",
```

Comment:
Can we use 0.6.0-alpha.xxx?

Reply:
Yes, I can update the SDK version next week. There's no 0.6.0-alpha version now.